jsonrpc/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@sourcegraph/vscode-jsonrpc",
 	"description": "A json rpc implementation over streams",
-	"version": "3.3.0-alpha.1",
+	"version": "3.3.0-alpha.1.sourcegraph.2",
```

Comment:
> latest upstream version is alpha.2

Right, but we haven't merged in alpha.2 changes yet. I briefly tried this and the merge conflict was non-trivial due to the rewrite of the platform independent change upstream, so I decided not to do that now.